pkg/ccl/crosscluster/logical/logical_replication_writer_processor.go

```diff
@@ -56,6 +56,22 @@ var flushBatchSize = settings.RegisterIntSetting(
 	settings.NonNegativeInt,
 )
 
+var writerWorkers = settings.RegisterIntSetting(
+	settings.ApplicationLevel,
+	"logical_replication.consumer.flush_worker_per_proc",
+	"the maximum number of workers per processor to use to flush each batch",
+	32,
+	settings.NonNegativeInt,
+)
+
+var minChunkSize = settings.RegisterIntSetting(
+	settings.ApplicationLevel,
+	"logical_replication.consumer.flush_chunk_size",
+	"the minimum number of row updates to pass to each flush worker",
```

Comment:
nit: s/row updates/ kvs/r

setting descriptions gotta be future proof for column families :)

Reply:
Eh, they are "row updates" in that they're an update to a part of a row. Plus it is non-public.